src/khoj/routers/api_chat.py

```diff
@@ -132,6 +137,60 @@ def chat_history(
     return {"status": "ok", "response": meta_log}
 
 
+@api_chat.get("/share/history")
+def chat_public_history(
```

Comment:
Should we simplify the endpoint to get /share?

```suggestion
@api_chat.get("/share")
def get_shared_chat(
```

Reply:
I think it makes sense to leave it as `/history` for nice parity with the private method. Can update the name.